lib/client.js

```diff
@@ -35,7 +57,7 @@ class Client {
   query(options) {
     const opc = this._config.parseQuery(options);
     if (!helper.config.validate(opc)) {
-      return Promise.reject('Invalid options');
+      return Promise.reject(new Error('Invalid options'));
```

Comment:
This is a change that it could be affect to the consumers if they expect an string through reject of the promise.